nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -262,7 +312,7 @@ Nav2Panel::Nav2Panel(QWidget * parent)
 
   accumulated_nav_through_poses_->assignProperty(
     navigation_mode_button_, "text",
-    "Start Waypoint Following");
+    "antp: Start Waypoint Following");
```

Comment:
what does "antp" mean? I would leave it unturned..

Reply:
This was for debugging. As far as I can tell, I can not figure out how to trigger this line of code.